broker-daemon/broker-rpc/orderbook-service/get-orderbook.js

```diff
@@ -25,6 +35,9 @@ async function getOrderbook ({ params, logger, orderbooks }, { GetOrderbookRespo
     const datetime = nano.toISOString(currentTime)
     const orders = await orderbook.all()
 
+    // limit is passed as an integer with default protobuf value of '0', so '0' implies limit was omitted
+    const limit = (params.limit === '0') ? DEFAULT_LIMIT : Big(params.limit)
```

Comment:
this is kind of a bummer, but I guess there isn't a reason for someone to call this endpoint with a `limit` of 0. The other way to achieve this is with [`oneof`](https://developers.google.com/protocol-buffers/docs/proto3#oneof), although that may be overkill for this case.